Contest_API.md

```diff
@@ -770,6 +770,65 @@ The `countdown_pause_time` may be changed to indicate the approximate delay unti
 Countdown is resumed by setting a new `start_time` and resetting
 `countdown_pause_time` to `null`.
 
+In most cases downstream clients should use `/scoreboard` endpoint to get standings. If for some reason, it's not suitable, one
+can use `scoreboard_rules` property to ensure that calculated scoreboard would be the same, or at least report potential
+diverges. Scoreboard rule object must have a `string` property named `kind` and any additional properties, describing settings. 
```

Comment:
I would prefer if this was worded to caution against calculating your own scoreboard stronger. I.e. you should not calculate your own scoreboard unless you *have* to, and then a list of reasonable reasons why you might have to (such as resolving, or calculating hypothetical future states and such)

Reply:
Not sure I love the choice of "kind". Would something like "ruleset" be better maybe?